src/input_cp2k_mp2.F

```diff
@@ -274,6 +274,16 @@ SUBROUTINE create_ri_mp2(section)
       CALL section_add_keyword(section, keyword)
       CALL keyword_release(keyword)
 
+      CALL keyword_create(keyword, __LOCATION__, name="NUMBER_INTEGRATION_GROUPS", &
+                          description="Sets the number of integration groups of the communication scheme in RI-MP2."// &
```

Comment:
Maybe mention that for a value<=0 the group size will be automatically computed from the available memory.

Reply:
Ah right, I have accidentally removed that hint.